javascript/javascript2/week1/spirit-animal-name/spirit-animal-name-generator-index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="/spirit-animal-name/spirit-animal-name-generator.css"
```

Comment:
I have tried to run your program from Visual Studio Code and Live Server and it doesn't work. One issue is the path to the .css file . It should be like this
```suggestion
      href="spirit-animal-name-generator.css"
```